site2/docs/administration-pulsar-manager.md

```diff
@@ -114,69 +68,116 @@ or
 jwt.broker.token.mode=SECRET
 jwt.broker.secret.key=file:///path/broker-secret.key
 ```
+•	`backend.jwt.token`: token for the superuser. You need to configure this parameter during cluster initialization.   
+•	`jwt.broker.token.mode`: multiple modes of generating token, including PUBLIC, PRIVATE, and SECRET.  
+•	`jwt.broker.public.key`: configure this option if you use the PUBLIC mode.  
+•	`jwt.broker.private.key`: configure this option if you use the PRIVATE mode.  
+•	`jwt.broker.secret.key`: configure this option if you use the SECRET mode.  
+For more information, see [Token Authentication Admin of Pulsar](https://pulsar.apache.org/docs/en/security-token-admin/). 
 
-* Method 3: use Docker and enable token authentication.
+Docker command to add profile and key files mount.
 
-```
-export JWT_TOKEN="your-token"
-docker run -it -p 9527:9527 -p 7750:7750 -e REDIRECT_HOST=http://localhost -e REDIRECT_PORT=9527 -e DRIVER_CLASS_NAME=org.postgresql.Driver -e URL='jdbc:postgresql://127.0.0.1:5432/pulsar_manager' -e USERNAME=pulsar -e PASSWORD=pulsar -e LOG_LEVEL=DEBUG -e JWT_TOKEN=$JWT_TOKEN -v $PWD:/data apachepulsar/pulsar-manager:v0.2.0 /bin/sh
+```bash
+docker pull apachepulsar/pulsar-manager:v0.2.0
+docker run -it \
+    -p 9527:9527 -p 7750:7750 \
+    -v /your-path/application.properties:/pulsar-manager/pulsar-
+manager/application.properties
+    -v /your-path/private.key:/pulsar-manager/private.key
```

Comment:
@fanla2021 seems that you do not incorporate @tuteng's suggestions (lack of \)?

Reply:
@Anonymitaet @tuteng I think the current docker quick start configuration is reasonable, and the complex configuration can be done by referring to the configuration notes below.